### ISSUE TITLE
clipto folder as a link

### ISSUE THREAD

**Issue by iomari** [clipto-pro/Desktop#144]
Greetings,
my ~/.config/clipto folder i a link to another location in my home folder.
It works fine that way but at times when I shut down or reboot my system, the link become a 240 byte file named "clipto" and the app won't load until I delete it and recreate the link.
Why does this happen? Does it mean that the clipto folder is unreliable as a link?

OS: Kubuntu 22.04

**Comment by atrashler** [comment]
Hmmmm. I have not tested it. I need to check such scenarios.

**Comment by iomari** [comment]
any luck

**Comment by atrashler** [comment]
No. I am in migration now and do not work on the project this year.

**Comment by iomari** [comment]
understood

**Comment by iomari** [comment]
anyone else have this issue or a solution?
